src/ApiStack.ts

```diff
@@ -60,4 +121,14 @@ export class ApiStack extends Stack {
     });
     return cert;
   }
+
+  private appIdStorage() {
+    const appIdStorage = new aws_dynamodb.Table(this, 'app-id-storage', {
+      partitionKey: { name: 'id', type: AttributeType.STRING },
+      billingMode: BillingMode.PAY_PER_REQUEST,
+      removalPolicy: RemovalPolicy.DESTROY,
```

Comment:
Willen we hier retain van maken in prod?

Reply:
Ja

Reply:
In config opnemen, prod niet weg, rest wel